.bazelrc

```diff
@@ -302,6 +302,7 @@ build:windows --action_env=TMPDIR
 build:windows --define signal_trace=disabled
 build:windows --define hot_restart=disabled
 build:windows --define tcmalloc=disabled
+build:windows --define wasm=disabled
```

Comment:
Could you also remove this from `ci/windows_ci_steps.sh` just to clean this up

Reply:
done